### PR TITLE
Migrate Demo Dvr Module

### PR DESCRIPTION
**Pull request by pp-pal**
## Typescript
- Update demo to use Dvr module client [#95](https://github.com/gl-zkPass/zkpass-sdk/pull/95)
- Remove NextJs demo [#95](https://github.com/gl-zkPass/zkpass-sdk/pull/95)
 
## Rust
- Update demo to use Dvr module client [#95](https://github.com/gl-zkPass/zkpass-sdk/pull/95)
- Remove query tool binary [#95](https://github.com/gl-zkPass/zkpass-sdk/pull/95)

### CODE REVIEW

File: rust/zkpass-core/src/dvr_app/dvr_helpers.rs [hunk 1]
```diff
@@ -2,20 +2,34 @@
...
  * Helpers for operations related to DVR token
  *
+ * Authors:
+ * Created at: July 1st 2024
+ * -----
+ * Last Modified: September 27th 2024
+ * -----
+ * Reviewers:
+ *
+ * ---
```
Comment (zulamdat) [inline rust/zkpass-core/src/dvr_app/dvr_helpers.rs:12]: @pp-pal kayanya bagian ini bisa di remove aja deh di file2nya

File: rust/zkpass-core/src/socket/errors.rs [hunk 2]
```diff
@@ -0,0 +1,326 @@
+/*
```
Comment (zulamdat) [inline rust/zkpass-core/src/socket/errors.rs:1]: @pp-pal perlukah directory socket dan logger jg kita copy ke repo ini?

Comment (pp-pal) [reply rust/zkpass-core/src/socket/errors.rs]: removed https://github.com/gl-zkPass/zkpass-sdk/pull/95/commits/8b22b83f66925cc95020d2258d8f2a31f6819f24

File: rust/zkpass-core/src/dvr_app/dvr_helpers.rs (hunk 1 above)
Comment (pp-pal) [reply rust/zkpass-core/src/dvr_app/dvr_helpers.rs]: updated https://github.com/gl-zkPass/zkpass-sdk/pull/95/commits/8b22b83f66925cc95020d2258d8f2a31f6819f24

**Review by WilliamhGDP — APPROVED** [review]

**Review by zulamdat — APPROVED** [review]

**Pull request merged**